crates/matrix-sdk-sqlite/src/event_cache_store.rs

```diff
@@ -465,6 +465,8 @@ impl EventCacheStore for SqliteEventCacheStore {
         key: &str,
         holder: &str,
     ) -> Result<bool> {
+        let _timer = timer!("method");
```

Comment:
Are we sure we want to put `method` everywhere?

Reply:
With `#[instrument]`, we already have all the information required. I really didn't know exactly what to put here. We are measuring the time for the whole method, I thought it was alright. Thought?